test/config.js

```diff
@@ -193,6 +176,117 @@ test('valid serviceName => active', function (t) {
   t.end()
 })
 
+test('serviceName defaults to package name', function (t) {
+  // TODO:
```

Comment:
Are these TODO's still relevant?

Reply:
Yeah, do you have any thoughts on it? Should those requires be moved up? Should I use already promisified modules?

Reply:
No I think it's fine to just use the normal modules. This also means we don't have to depend on two different versions of mkdirp.

It might be cleaner to move the require's up top. I get why you put them here, but to follow our thing about how to order the require statements, it's probably better to be consistent unless there's a specific reason.

Reply:
fixed